### ISSUE TITLE
my trust wallet is empty after i connected wallet on cowswapltd.finance

### ISSUE THREAD

**Issue by hybebe** [hybebe/john#1]
did my wallet get hacked?

**Comment by whetherlove** [comment]
> did my wallet get hacked?

Yes it was hacked I had lost some funds due to an airdrop scam too and it fucked me up luckily I got introduced to this hacker who helped me recover what was stolen from me hit him up on telegram:: https://t.me/Blackhathacker110

**Comment by Bendy127** [comment]
So sorry about what happened same thing happened to me my wallet got hacked and about 0.9 btc was stolen thanks to Mr John a whitehathacker who helped me recover all of it message him on telegram too t.me/Whitehathacker217

**Comment by Bendy127** [comment]
> > did my wallet get hacked?
> 
> Yes it was hacked I had lost some funds due to an airdrop scam too and it fucked me up luckily I got introduced to this hacker who helped me recover what was stolen from me hit him up on telegram:: https://t.me/Blackhathacker110

So sorry about what happened same thing happened to me my wallet got hacked and about 0.9 btc was stolen thanks to Mr John a whitehathacker who helped me recover all of it message him on telegram too t.me/Whitehathacker217
